media/com_fabrik/js/tipsBootStrapMock.js

```diff
@@ -230,6 +244,13 @@ define(['jquery', 'fab/fabrik'], function (jQuery, Fabrik) {
                         .css(tp)
                         .addClass(placement)
                         .addClass('in');
+                    if ( (tippos.includes("top") || tippos.includes("bottom")) 
+                        && xpos > 0 && xpos > leftpos+15 && xpos < leftpos+maxwidth-15 ) {
```

Comment:
If tip is positioned on top or bottom - and xpos has been set by mouseover cursor position - and xpos is within the left and right boundaries of the popover (+-15px), then move arrow to xpos (mouseover cursor) position.

Reply:
...otherwise the arrow will be positioned like it had been,